### ISSUE TITLE
Position id3v2 tag in versions 0.3.5 and 0.3.7

### ISSUE THREAD

**Issue by dogwatch** [ffalt/jamp3#95]
Where is the mistake?

`const r5 = await mp3_035.read(fn, {id3v2: true});`
`const r7 = await mp3_037.read(fn, {id3v2: true});`

`console.log(r5.id3v2.end);  // 394816`
`console.log(r7.id3v2.end);  // 392816`

**Comment by ffalt** [comment]
In 0.3.7 I fixed a position bug, I forgot to document. it is documented [now](https://github.com/ffalt/jamp3/blob/master/CHANGELOG.md#0.3.7)

**Comment by dogwatch** [comment]
Note the size and position of the tag.
Tag written and read in version 0.3.9

```
{ size: 6174747,
  id3v2:
   { id: 'ID3v2',
     start: 0,
     end: 300213,
     head:
      { ver: 3,
        rev: 0,
        size: 302203,
        valid: true,
```

**Comment by ffalt** [comment]
this is far too few information for me to do anything. i can only guess, but i won't.

**Comment by dogwatch** [comment]
```
await mp3.removeTags(fn, {id3v2: true});
console.log((await fs.stat(fn)).size);  // 5 874 534

await id3v2.write(fn, (new ID3V24TagBuilder()).buildTag(), 3, 0);
console.log((await fs.stat(fn)).size);  // 5 876 544

await mp32.removeTags(fn, {id3v2: true});
console.log((await fs.stat(fn)).size);  // 5 876 534
```

**Comment by dogwatch** [comment]
I think the problem here is:
https://github.com/ffalt/jamp3/blob/ff77098bab90dfcaece63194b742ab3e5867b827/src/lib/id3v2/id3v2_writer.ts#L504

**Comment by ffalt** [comment]
yes, thanks! you found it. padding was not removed. Fixed in [0.3.10](https://github.com/ffalt/jamp3/blob/master/CHANGELOG.md#0.3.10): everything before the first audio frame is stripped when removing id3v2. this is also now part of the [tests](https://github.com/ffalt/jamp3/blob/5aa37c4443c21d1a94c0e6ad4c51ec8dbb16c33d/test/mp3/test.ts#L264).
